bundles/org.openhab.binding.avmfritz/src/main/resources/ESH-INF/thing/bridge-types.xml

```diff
@@ -7,9 +7,12 @@
 	<!-- Supported FRITZ!Box and FRITZ!Powerline -->
 	<bridge-type id="fritzbox">
 		<label>FRITZ!Box</label>
-		<description>This bridge represents a FRITZ!Box.</description>
+		<description>A FRITZ!Box router.</description>
 
 		<channels>
+			<channel id="incoming_call" typeId="incoming_call"/>
```

Comment:
See my comment above about activation of call monitor. So this channels should be added dynamicly if call monitor is active